### ISSUE TITLE
RUN THE CODE ON COLAB

### ISSUE THREAD

**Issue by iremonur** [ZhangGongjie/Meta-DETR#44]
Hi, 
I would like to run the code on COLAB, because I have only one GPU on my computer, but I suppose COLAB also has only one GPU. Is there any way to run the code on COLAB with multiple GPUs? Or any recommendation to run the code with multiple GPUs on an alternative platform? 
I would be glad if you help me with this.

**Comment by ZhangGongjie** [comment]
I haven't used COLAB too much. Honestly I don't know how to use multiple GPUs on COLAB. Sorry about it.

Using only 1 GPU can produce certain performance with our pre-trained model weights.

To use one GPU to improve performance, you may try the following methods:
(1) Use amp (Automatic Mixed Precision).
(2) Perform 8 feedforwards, then accumulate the gradients, and then perform back propagation once. This should be theoretically equivalent to our fine-tuning setups.

Thank you :)
